src/js/main.js

```diff
@@ -30,3 +30,7 @@ import '../components/modal/modal.dom';
 import '../components/timeout-modal/timeout-modal.dom';
 import '../components/timeout-panel/timeout-panel.dom';
 import '../components/video/video.dom';
+import '../components/charts/chart';
+import '../components/charts/chart-options';
```

Comment:
We should only need to import `chart.js` here as the options and line-chart options are imported in turn by `chart.js`